libraries/rush-lib/src/utilities/prompts/SearchListPrompt.ts

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import type { Interface } from 'readline';
+import _ from 'lodash';
+import colors from 'colors/safe';
+
+// Modified from the choice list prompt in inquirer:
+// https://github.com/SBoudrias/Inquirer.js/blob/inquirer%407.3.3/packages/inquirer/lib/prompts/list.js
```

Comment:
We cannot in this specific case because of the way we have to manually track the `_selected` index of values. We are using a different method to track what that value is based on how filtering occurs and what keys are pressed during the track. See the onkeyevents for more details.